src/WebJobs.Extensions.DurableTask/EventIndexDateMapping.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class EventIndexDateMapping
```

Comment:
I didn't check, but does any other class than `DurableOrchestrationClient` use `EventIndexDateMapping`? If not, instead of adding it as it's own file, can we make it a private inner class of `DurableOrchestrationClient`.

Reply:
Yes, great point. I will move it. Thank you very much!